common/src/main/java/com/zegelin/cassandra/exporter/collector/StorageServiceMBeanMetricFamilyCollector.java

```diff
@@ -91,6 +118,23 @@ private StorageServiceMBeanMetricFamilyCollector(final StorageServiceMBean stora
             metricFamilyStreamBuilder.add(new GaugeMetricFamily("cassandra_token_ownership_ratio", null, ownershipMetricStream));
         }
 
+        {
+            String localIP = storageServiceMBean.getHostIdToEndpoint().get(storageServiceMBean.getLocalHostId());
+            final Stream<NumericMetric> ownershipMetricStream = metadataFactory.keyspaces().stream()
+                    .filter(keyspace -> !excludedKeyspaces.contains(keyspace))
+                    .flatMap(keyspace -> {
+                        try {
```

Comment:
In order to increase readability for this method, what do you think about extracting this block into a dedicated method? I imagine it could return a `Stream<...>`

Reply:
I agree, and I'll remember this. But if you look further down (see `effectiveOwnership(keyspace)`) in the code you'll see that's not how the code is written, so for now I'd keep the same code style.